src/shared-styles/_baseline.scss

```diff
@@ -0,0 +1,38 @@
+@use "~modularscale-sass/stylesheets/modularscale";
+@use "scale";
+
+//----------------------------------------------------------------------------------------------------------------------
+// Baseline grid lines height.
+//
+// Every spacing metric should be based on this.
+// All commented sizes assume a scale base of 16px.
+//
+$_baseline-unit: scale.$base * 0.375; // 6px
+
+//----------------------------------------------------------------------------------------------------------------------
+// Spacing values determined by the global baseline grid.
+//
+@function _baseline-unit($factor) {
+  @return $_baseline-unit * $factor;
+}
+
+$size-1: _baseline-unit(1); // 6px
```

Comment:
Could make these private and expose those currently required via names of convenience. It raises the question of what those names should be. Some options are:
1. keep `size` variables public and just use these. Not a bad option, and it means we don't have to keep coming up with sensible convenience names whenever a new `size` value needs making public
1. make `size` variables private and use the original names for convenience public variables or
1.  make `size` variables private and use newly suggested names for convenience public variables or
1.  make `size` variables private and use Reviewer's names for convenience public variables if they would work sensibly here too.

Right now options 1 and 4 look the best, with a slight preference for 1.